generate-go.sh

```diff
@@ -1,5 +1,4 @@
-#!/bin/bash
-
+#!/usr/bin/env bash
```

Comment:
Why change it?

Reply:
It used to be /usr/bin/env bash, and I created this branch before it is changed to /bin/bash (and rebased before file pr), I will correct this.